mlir/test/xmir/e2e/resnet50_blk.fail.mlir

```diff
@@ -0,0 +1,45 @@
+// RUN: rocmlir-driver -host-pipeline partition,highlevel -targets gfx900 %s | rocmlir-gen -ph -print-results -rand_type float -rand 1 -fut resnet50 --verifier clone - | rocmlir-driver -host-pipeline xmodel -kernel-pipeline full | not xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_async_runtime%shlibext --entry-point-result=void 2>&1 | FileCheck %s
```

Comment:
Could we make this a much more targeted test? Run this through almost all of the pipeline and then just `RUN: xmir-runner %s`

Reply:
Good idea.